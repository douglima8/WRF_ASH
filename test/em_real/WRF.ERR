@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@    WARNING: REMOTE HOST IDENTIFICATION HAS CHANGED!     @
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
IT IS POSSIBLE THAT SOMEONE IS DOING SOMETHING NASTY!
Someone could be eavesdropping on you right now (man-in-the-middle attack)!
It is also possible that a host key has just been changed.
The fingerprint for the ECDSA key sent by the remote host is
f1:7f:a2:a3:b5:70:59:27:1a:37:7c:0c:95:d9:b0:61.
Please contact your system administrator.
Add correct host key in /home/rizza/.ssh/known_hosts to get rid of this message.
Offending ECDSA key in /home/rizza/.ssh/known_hosts:7
  remove with: ssh-keygen -f "/home/rizza/.ssh/known_hosts" -R socrate12
Password authentication is disabled to avoid man-in-the-middle attacks.
Keyboard-interactive authentication is disabled to avoid man-in-the-middle attacks.
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@    WARNING: REMOTE HOST IDENTIFICATION HAS CHANGED!     @
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
IT IS POSSIBLE THAT SOMEONE IS DOING SOMETHING NASTY!
Someone could be eavesdropping on you right now (man-in-the-middle attack)!
It is also possible that a host key has just been changed.
The fingerprint for the ECDSA key sent by the remote host is
f1:7f:a2:a3:b5:70:59:27:1a:37:7c:0c:95:d9:b0:61.
Please contact your system administrator.
Add correct host key in /home/rizza/.ssh/known_hosts to get rid of this message.
Offending ECDSA key in /home/rizza/.ssh/known_hosts:1
  remove with: ssh-keygen -f "/home/rizza/.ssh/known_hosts" -R socrate11
Password authentication is disabled to avoid man-in-the-middle attacks.
Keyboard-interactive authentication is disabled to avoid man-in-the-middle attacks.
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@    WARNING: REMOTE HOST IDENTIFICATION HAS CHANGED!     @
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
IT IS POSSIBLE THAT SOMEONE IS DOING SOMETHING NASTY!
Someone could be eavesdropping on you right now (man-in-the-middle attack)!
It is also possible that a host key has just been changed.
The fingerprint for the ECDSA key sent by the remote host is
f1:7f:a2:a3:b5:70:59:27:1a:37:7c:0c:95:d9:b0:61.
Please contact your system administrator.
Add correct host key in /home/rizza/.ssh/known_hosts to get rid of this message.
Offending ECDSA key in /home/rizza/.ssh/known_hosts:8
  remove with: ssh-keygen -f "/home/rizza/.ssh/known_hosts" -R socrate13
Password authentication is disabled to avoid man-in-the-middle attacks.
Keyboard-interactive authentication is disabled to avoid man-in-the-middle attacks.
 starting wrf task           11  of           64
 starting wrf task            3  of           64
 starting wrf task            4  of           64
 starting wrf task           12  of           64
 starting wrf task           14  of           64
 starting wrf task            7  of           64
 starting wrf task            1  of           64
 starting wrf task            8  of           64
 starting wrf task            6  of           64
 starting wrf task           10  of           64
 starting wrf task           15  of           64
 starting wrf task            0  of           64
 starting wrf task            5  of           64
 starting wrf task           13  of           64
 starting wrf task            9  of           64
 starting wrf task            2  of           64
 starting wrf task           37  of           64
 starting wrf task           33  of           64
 starting wrf task           41  of           64
 starting wrf task           42  of           64
 starting wrf task           55  of           64
 starting wrf task           50  of           64
 starting wrf task           35  of           64
 starting wrf task           53  of           64
 starting wrf task           40  of           64
 starting wrf task           62  of           64
 starting wrf task           34  of           64
 starting wrf task           57  of           64
 starting wrf task           38  of           64
 starting wrf task           45  of           64
 starting wrf task           46  of           64
 starting wrf task           63  of           64
 starting wrf task           51  of           64
 starting wrf task           56  of           64
 starting wrf task           18  of           64
 starting wrf task           25  of           64
 starting wrf task           48  of           64
 starting wrf task           58  of           64
 starting wrf task           32  of           64
 starting wrf task           19  of           64
 starting wrf task           36  of           64
 starting wrf task           39  of           64
 starting wrf task           54  of           64
 starting wrf task           44  of           64
 starting wrf task           52  of           64
 starting wrf task           43  of           64
 starting wrf task           49  of           64
 starting wrf task           47  of           64
 starting wrf task           16  of           64
 starting wrf task           20  of           64
 starting wrf task           61  of           64
 starting wrf task           31  of           64
 starting wrf task           23  of           64
 starting wrf task           17  of           64
 starting wrf task           60  of           64
 starting wrf task           22  of           64
 starting wrf task           26  of           64
 starting wrf task           30  of           64
 starting wrf task           24  of           64
 starting wrf task           27  of           64
 starting wrf task           28  of           64
 starting wrf task           59  of           64
 starting wrf task           29  of           64
 starting wrf task           21  of           64
