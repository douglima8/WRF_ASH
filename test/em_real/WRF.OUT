Warning: no access to tty (Bad file descriptor).
Thus no job control in this shell.
Running on host socrate10.cluster.it
Time is Mon Sep 27 16:04:00 CEST 2021
Directory is /cantiere/rizza/WRFCHEM/WRFV361/test/em_real
This jobs runs on the following processors:
socrate10 socrate10 socrate10 socrate10 socrate10 socrate10 socrate10 socrate10 socrate10 socrate10 socrate10 socrate10 socrate10 socrate10 socrate10 socrate10 socrate11 socrate11 socrate11 socrate11 socrate11 socrate11 socrate11 socrate11 socrate11 socrate11 socrate11 socrate11 socrate11 socrate11 socrate11 socrate11 socrate12 socrate12 socrate12 socrate12 socrate12 socrate12 socrate12 socrate12 socrate12 socrate12 socrate12 socrate12 socrate12 socrate12 socrate12 socrate12 socrate13 socrate13 socrate13 socrate13 socrate13 socrate13 socrate13 socrate13 socrate13 socrate13 socrate13 socrate13 socrate13 socrate13 socrate13 socrate13
This job has allocated 64 nodes
